qa/L0_java_sequence_batcher/Simple.java

```diff
@@ -0,0 +1,671 @@
+// Copyright (c) 2021-2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+//
```

Comment:
This is a change of this from the original Simple.java code right? Can we change the name to something like simple_sequence.java or similar? Also change copyright

Reply:
But also, maybe it's better to just have one Simple.java that calls a separate function unique for that test.

Reply:
That's correct.

We could probably wrap a lot of the functionality in Simple, and have each test call a child class of Simple specific to it with those functions overridden. Would that solution be ideal? Are you doing L0_java_infer differently; if not, let me know what customizations you would need for that.

Reply:
@jbkyang-nvi As discussed, I renamed Simple and added comments to describe what is boilerplate vs custom code.

Once we have L0_java_sequence_batcher, L0_java_infer, and L0_java_memory_growth merged, I could look into turning them into one test framework class file to avoid repeating boilerplate code, if we think it makes sense.